programs/budget_api/src/budget_expr.rs

```diff
@@ -28,6 +32,23 @@ pub struct Payment {
     pub to: Pubkey,
 }
 
+/// The account constraints a Condition would wait on.
```

Comment:
Output register of some previous instruction?

Reply:
Yes, that's a way to describe it in ABI terminology.